sample.env

```diff
@@ -20,3 +20,4 @@ MAINTENANCE_MODE=false
 # date format must be HH:mm DD/MM/YYYY, only used if MAINTENANCE_MODE is true
 SERVICE_AVAILABLE_DATE=14:30 02/03/2019
 APP_VERSION=local
+NOTIFY_API_KEY=notify-test-key-replace-with-real-one-in-1password
```

Comment:
Should we be saying `1password` here as this is publicly available? Maybe just use `xxxxxxx-xxxxxxx-xxxxx-xxxxx` or similar

Reply:
I'll remove the 1password section but it's not really a security concern as people would still need to know our 1password credentials.